lib/commands/actions.js

```diff
@@ -29,7 +29,10 @@ commands.longPressKeyCode = async function (keycode, metastate = null) {
 };
 
 commands.getOrientation = async function () {
-  let orientation = await this.bootstrap.sendAction("orientation", {});
+    let params = {
```

Comment:
This indentation is two spaces too far. Can you run eslint locally (`gulp eslint`)?